docs/decisions/0002-lti-1p3-variables.rst

```diff
@@ -0,0 +1,161 @@
+The roles claim takes in a list of LTI 1.3 compliant roles from http://www.imsglobal.org/spec/lti/v1p3/#role-vocabularies.
+The claim name is: https://purl.imsglobal.org/spec/lti/claim/roles.
+
+The mappings from edX platform are shown in the table below:
+
+.. list-table::
+   :widths: auto
+   :header-rows: 1
+
+   * - Open edX role
+     - LTI 1.3 Roles included
+     - Reasoning
+   * - guest
+     - Empty
+     - Guests are users not logged in, they shoudn't be able to access LTI content.
+   * - student
+     - 'http://purl.imsglobal.org/vocab/lis/v2/institution/person#Student'
+     - Students only have permission to view and interact with the LTI content.
+   * - instructor
+     - 'http://purl.imsglobal.org/vocab/lis/v2/institution/person#Instructor'
+       'http://purl.imsglobal.org/vocab/lis/v2/institution/person#Student'
+     - Instructor have both instructor and student access to a tool. No admin permissions.
+   * - staff
+     - 'http://purl.imsglobal.org/vocab/lis/v2/system/person#Administrator'
+       'http://purl.imsglobal.org/vocab/lis/v2/institution/person#Instructor'
+       'http://purl.imsglobal.org/vocab/lis/v2/institution/person#Student'
+     - A staff user should be able to modify the tool settings and have full access to the content and settings.
+
+Resource link
+~~~~~~~~~~~~~
+
+This claim has properties for the resource link from which the launch message occurs. It represents a unique place in the course where the LTI launch happens.
+
+.. list-table::
+   :widths: auto
+   :header-rows: 1
+
+   * - Claim variable
+     - Block location (on course context).
+     - Reasoning
+   * - id
+     - Block location (on course context).
+     - The resource link id needs to be a identifier unique to the deployment.
+       In the Open edX platform, we already have an analog that is unique to the entire instance: block location.
+   * - title
+     - Display name (either from block or parent block)
+       Empty if not available/applicable.
+     - We use the `Display name` attribute to provide a descriptive title to the LTI tool.
+       In case this isn't available in the launch context, this parameter isn't sent.
+   * - description
+     - Not implemented.
+     - This is an optional field.
+
+Reference: http://www.imsglobal.org/spec/lti/v1p3/#resource-link-claim-0
+
+User claims
+~~~~~~~~~~~
+The user data claims are identifiers passed through the LTI message to send user data to the LTI tool.
+
+.. list-table::
+   :widths: auto
+   :header-rows: 1
+
+   * - Claim variable
+     - Open edX
+     - Reasoning
+   * - sub
+     - external user id (with the `lti` type)
+     - Using the id provided by the external_user_id from openedx core.
+       See https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/external_user_ids/docs/decisions/0001-externalid.rst
+   * - name
+     - User full name
+     - User full name as defined in the spec, gated by a feature flag to avoid leaking PII to external tools.
```

Comment:
The feature flag defaults on or off?

Reply:
If should probably default to `off`.
This is currently not implemented through a feature flag, and relies on the `lti-configuration` service from the LTI 1.1 implementation. We can turn it into a flag, since there's not that much documentation on the LTI Configuration service.